site/content/en/docs/manual/advanced/serverless-tutorial.md

```diff
@@ -898,6 +898,59 @@ $ docker logs nuclio-nuclio-pth.shiyinzhang.iog
 
 </details>
 
+### Problem with running nuctl on Windows 10
+
+If you encounter the problem with running `nuctl` on Windows 10, you can do the following:
+
+1. Install [Ubuntu 18.04 from Microsoft store][ubuntu-18-04-microsoft-store].
+
+1. Enable integration for Ubuntu-18.04 in the settings `Docker Desktop` in the `Resourcrces` `WSL integration` tab
+   (it is understood that `Docker` and `WSL 2` are installed on windows):
+
+   ![Docker WSL integration Ubuntu 18.04](/images/docker_wsl_integration.jpg)
+
+1. Install `curl` on Ubuntu with use command:
+   ```sh
+   sudo apt install curl
+   ```
+1. Installation `nuctl` on Ubuntu, using the command from [manuals on nuclio CLI][nuctl-manual]:
+   ```sh
+   curl -s https://api.github.com/repos/nuclio/nuclio/releases/1.5.16 \
```

Comment:
First of all it isn't a good idea to use curl and wget in the same command. Both commands are using for the same purpose. I will recommend to simplify it for common instructions. In general we should recommend using "linux" instructions. Better to contribute into the "linux" part to make it better.

```
You have to install nuctl command line tool to build and deploy serverless functions. Download version 1.5.16. It is important that the version you download matches the version in docker-compose.serverless.yml After downloading the nuclio, give it a proper permission and do a softlink

sudo chmod +x nuctl-<version>-linux-amd64
sudo ln -sf $(pwd)/nuctl-<version>-linux-amd64 /usr/local/bin/nuctl
```